### PR TITLE
Challenge2 2

### PR DESCRIPTION
**Pull request by Boo171717**
@lpaglione
@pedroha
@vijithassar

This is my revised version.

### CODE REVIEW

**Comment by lpaglione** [comment]
@Boo171717  looks good. Nice use of comments!

**Comment by Boo171717** [comment]
@lpaglione 
Hi I'm done with this hw.

**Comment by lpaglione** [comment]
bitmoji

Looks good! Go ahead and merge this branch @Boo171717

File: Fall-2017-JS02/Challenge2/index.html [hunk 1]
```diff
@@ -1,10 +1,24 @@
...
-    </script>
-  </body>
+<body>
+         <script src = "https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <ins style="display:block; text-align:center;"
+  ></ins>
+<script>
+</script><h2>TextBox value : <label id="msg"></label></h2>
+
+<div style="padding:16px;">
```
Comment (pedroha) [inline Fall-2017-JS02/Challenge2/index.html:10]: the padding could go into a CSS file

File: Fall-2017-JS02/Challenge2/index.html [hunk 2]
```diff
@@ -1,10 +1,24 @@
...
-    <link rel="stylesheet" type="text/css" href="project.css">
-  </head>
-  <body>
-    <div class="greeting"> Hello world!!! </div>
-    <script src="project.js">
-    </script>
-  </body>
+<body>
+         <script src = "https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <ins style="display:block; text-align:center;"
```
Comment (pedroha) [inline Fall-2017-JS02/Challenge2/index.html:5]: You can probably use a `<div>` tag instead of an `<ins>` tag, maybe with some CSS

File: Fall-2017-JS02/Challenge2/project.css [hunk 3]
```diff
@@ -1,7 +1,14 @@
 div.greeting {
-  color: rgb(0, 0, 250);
+  color: rgb(0, 0, 250); /*making color*/
```
Comment (pedroha) [inline Fall-2017-JS02/Challenge2/project.css:2]: I like that you added comments to most things. Here, it might be a better comment to say what color is that mysterious `rgb(0, 0, 25);`

File: Fall-2017-JS02/Challenge2/project.css [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
+  font-family: "Times New Roman"; /*making the font family times new roman*/
+  font-size: 100px; /*making the font size 100px*/
 }
 body {
-  background-color: rgb(0, 250, 0);
-
+  background-color: rgb(0, 250, 0); /* background color green*/
+}
+ div {width: 320px;padding: 10px;border: 5px solid gray;margin: 0;} /*input[type=text] {
```
Comment (pedroha) [inline Fall-2017-JS02/Challenge2/project.css:9]: it's good practice to list all the CSS properties in separate lines.

File: Fall-2017-JS02/Challenge2/project.js [hunk 5]
```diff
@@ -1 +1 @@
-console.log("Hi there!")
+console.log("Hi there") //making the console state what is inside the quotes/
```
Comment (pedroha) [inline Fall-2017-JS02/Challenge2/project.js:1]: I like comments that are more of human dialogue, `making the console tell us a greeting`
